Non-Interactive-Protocol.md

```diff
@@ -0,0 +1,173 @@
+local fee-estimator and ensure it's well-connected to the tx-relay network.
+
+We recommend to not react on any mempool event as otherwise it would be a source of
+cross-layer mapping.
+
+We recommend to not sign local transactions until their broadcast is required
+to avoid their misuage in case of site compromise, assuming an external signer.
+
+# Funding Phase
+
+A party always starts the protocol in the funding phase. The funding phase can
+terminate by either a transaction to the execution phase or a protocol abort.
+
+## Requirements
+
+A node:
+	- if a counterparty's funding signature is received:
+		- MUST finalize the funding transaction with a local `SIGHASH_ALL` signature
+		- MUST broadcast the funding transaction
+
+A node:
+	- if a funding signature has been sent or a funding transaction broadcast and there is no confirmation after 6 blocks:
+		- MUST spend the change output with a CPFP increasing package feerate to confirm soonly
+		- MUST start a bumping timer of length `congestion_delay`
+		- if `congestion_delay` expires:
+			- MUST spend the change output with a CPFP increasing package feerate to confirm soonly
+			- MUST replace according to BIP 125
+			- MUST reschedule timer for next bump
+		- if `contract_maturity_bound` is 20 blocks ahead from local tip:
+			- MUST spend the funding input with a higher-feerate transaction/absolute fee than any previously propagated package feerate
+			- MUST start a bumping timer of length `security_delay`
+			- MUST disable `congestion_delay` timer
+		- if `security_delay` expires:
+			- MUST spend the funding input with a higher-feerate transaction/absolute fee than any previously propagated package feerate
+			- MUST replace according to BIP 125
+			- MUST reschedule timer for next bump
+		- MUST NOT spend more in bumping fees than its collateral input value
+
+A node:
+	- if the funding transaction confirms:
+		* MUST track spend of the CET output
+		* MUST advances its off-chain state to the execution phase
+		* MUST note its change output for latter spend
+
+A node:
+	- if the collateral double-spend confirms:
+		* MUST abort the DLC and prevent further interactions with counterparty
+
+## Rationale
+
+Funding transaction must be confirmed to consider the off-chain state initialized. As mempools
+congestion may fluctuate between fees negotiation and transaction propagation, party must adjust
+unilaterally feerate according to their confirmation preferences. A party may choose to wait
+for mempools draining before to adjust the feerate to avoid a fee waste as block height of 
+transition to execution phase doesn't matter, assuming it's before `contract_maturity_bound`.
+
+Reaching near `contract_maturiy_bound` requires to confirm the funding transaction quickly as it's
+now a security matter. After oracle signature release, a malicious party could try to double-spend
+an unconfirmed funding transaction to avoid loosing its collateral.
```

Comment:
I'm not sure I fully agree with that. In my view, as long as you don't have a confirmed fund transaction, you shouldn't consider yourself as being in the contract. Your counter-party double spending the fund tx might be a malicious behavior, but from what I understand it might also be genuine. So I'm not sure why you label that as a "security" issue.

Reply:
I think we agree on "you shouldn't consider yourself as being in the contract" as I made mandatory : "if the funding transaction confirms MUST advances its off-chain state to the execution phase". 

> Your counter-party double spending the fund tx might be a malicious behavior, but from what I understand it might also be genuine.

Without interaction with your counterparty how do you know if it's a malicious or genuine behavior :) ? In case of doubt, assume the worst-case scenario.